specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 2029:
```
  cvr_table: TYPE =
    sequence[[county, cvr]]
```
We presumably want to index this by ballot_manifest_info, so perhaps that should actually be managed
separately from `cvr`, or just added separately in here if that helps with the indexing (in which case
we can drop `county`.